### DISCUSSION TITLE
Will the “Web Integrity API” come to ungoogled-chromium?

### DISCUSSION THREAD

**Discussion by Mick-DE** [Eloston/ungoogled-chromium#2434] [General]
Firefox does have its quirks, but at least nobody there tries [to build in DRM for websites](https://translate.google.com/website?sl=de&tl=en&hl=en&prev=search&u=https://github.com/RupertBenWiser/Web-Environment-Integrity/blob/main/explainer.md) so that websites can disable your adblocker to protect their business model.

**"Users often depend on websites trusting the client environment they run in."**
-Uh... no. The other way round. Users trust the client environment not to let websites do shit.

**"This trust may assume that the client environment is honest about certain aspects of itself, keeps user data and intellectual property secure, and is transparent about whether or not a human is using it."**
-Ooooh did you come up with a broken business model because you were too stupid to do some research on the realities of the environment you were going to poo hard on the lawn in? But that's unfortunate, dear startup founder, dear tech bro, dear VC burner.

**"This trust is the backbone of the open internet, critical for the safety of user data and for the sustainability of the website's business."**
- THIS is Google's optimization goal when they talk about the "open Internet": They want a guarantee that their "business models" work at the expense of the users and that they cannot defend themselves. And since you jerks were all stupid enough to use their browsers, now you can just do it like that.

https://blog.fefe.de/?ts=9a4452d8
https://github.com/RupertBenWiser/Web-Environment-Integrity/blob/main/explainer.md
https://arstechnica.com/gadgets/2023/07/googles-web-integrity-api-sounds-like-drm-for-the-web/

**Comment by nilslindemann** [comment]
Is this a question or a pamphlet? ;-)

Anyway, I hope they can cut out this technology from UGC, or allow disabling it. Firefox is no real alternative.

I am fine when websites close me out as a result of disabling it. They already do that with their pay-or-cookie walls, etc.

**Reply by PF4Public** [reply]
To be fair, this one is more modest compared to #2432.
